src/lib/SUI.js

```diff
@@ -609,6 +609,7 @@ export const GENDERS = [
   'mars stroke vertical',
   'mercury',
   'neuter',
+  'other',
```

Comment:
This array contains definitions for existing icons, please revert this change.

Reply:
ff96fdc should fix it